scripts/devops_tasks/setup_execute_tests.py

```diff
@@ -22,14 +22,16 @@
 # evaluating whether we want this or not.
 ALLOWED_RETURN_CODES = []
 
+MANAGEMENT_PACKAGE_IDENTIFIERS = ['mgmt', 'azure-cognitiveservices-personalizer']
```

Comment:
@lmazuel 

Are all cogservices generated? If so I'll just put `azure-cognitiveservices` and we'll handle all of them this way.

Reply:
To unblock release though, I'm just merging tonight. ETA for release tomorrow noon.